### ISSUE TITLE
I got an error when I added "background_location: ^0.8.1 " package to my project.

### ISSUE THREAD

**Issue by SarahGhorbani** [Almoullim/background_location#171]
I got this error when I added just the backgroud_location: ^0.8.1 package to my project: 

```
 C:\SDK\flutter_sdk\.pub-cache\hosted\pub.dartlang.org\background_location-0.8.1\android\src\main\kotlin\com\almoullim\background_location\BackgroundLocationService.kt: (23, 1): Class 'BackgroundLocationService' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult(p0: Int, p1: Array<(out) String!>, p2: IntArray): Boolean defined in io.flutter.plugin.common.PluginRegistry.RequestPermissionsResultListener
e: C:\SDK\flutter_sdk\.pub-cache\hosted\pub.dartlang.org\background_location-0.8.1\android\src\main\kotlin\com\almoullim\background_location\BackgroundLocationService.kt: (221, 5): 'onRequestPermissionsResult' overrides nothing
```

**Comment by Zeinab-Kouhkan** [comment]
Same as me

**Comment by devMehdiD** [comment]
thanks you

**Comment by Frostfire25** [comment]
Thank you, this helped resolve my issue.
 'BackgroundLocationService' is not abstract an d does not implement abstract member public abstract

**Comment by santhoshrao519** [comment]
getting
Class 'LocationPermissionManager' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult(p0: Int, p1: Array<(out)
 String!>, p2: IntArray): Boolean defined in io.flutter.plugin.common.PluginRegistry.RequestPermissionsResultListener

**Comment by MoralCode** [comment]
This seems to be resolved in #174  and is part of the tagged version 0.10.0. It is not yet on pub.dev though so you'll probably need to use the latest main branch for now.
